phys2bids/tests/test_txt.py

```diff
@@ -4,40 +4,42 @@
 from pytest import raises
 import sys
 import io
+import wget
 
 
 def test_read_header_and_channels():
-    test_filename = 'Test_belt_pulse_samefreq.txt'  # this file has a comment and extra tab
+    url = 'https://osf.io/sdz4n/download'
```

Comment:
Could you please add a comment that states this is the url to the file `test_filename`?